shimmingtoolbox/shim/sequencer.py

```diff
@@ -988,8 +1052,7 @@ def _optimize(optimizer: Optimizer, nii_mask_anat, slices_anat, shimwise_bounds=
 gl_shimwise_bounds = None
 
 
-def _worker_init(optimizer, nii_mask_anat, slices_anat, dilation_kernel, dilation_size, path_output,
-                 shimwise_bounds):
+def _worker_init(optimizer, nii_mask_anat, slices_anat, dilation_kernel, dilation_size, path_output, w_signal_loss, epi_te, shimwise_bounds):
```

Comment:
Samer comment as above, I don't think you need `w_signal_loss` and  `epi_te` in this function

Reply:
Keep it as it is since removing these two variables seems to give me errors